pkg/server/status.go

```diff
@@ -849,6 +862,44 @@ func (s *statusServer) Nodes(
 			return nil, grpcstatus.Errorf(codes.Internal, err.Error())
 		}
 	}
+
+	// get the derivatives here?
+	var queries []tspb.Query
+	for _, nodeStatus := range resp.Nodes {
+		for _, metric := range IoMetrics {
+			deriv := tspb.TimeSeriesQueryDerivative_NON_NEGATIVE_DERIVATIVE
+			query := tspb.Query{
+				Name:       "cr.node." + metric,
+				Sources:    []string{nodeStatus.Desc.NodeID.String()},
+				Derivative: &deriv,
+			}
+			queries = append(queries, query)
+		}
+	}
+	now := time.Now().UnixNano()
+	tsReq := tspb.TimeSeriesQueryRequest{
+		EndNanos:    now,
+		StartNanos:  now - int64(40*time.Second),
+		SampleNanos: int64(10 * time.Second),
+		Queries:     queries,
+	}
```

Comment:
doing something similar to the metric data provider here

Reply:
Regardless of whether this is the correct strategy, this is the correct way to query the time series system.  That said, I'm not sure its the correct strategy; let me think about that for a bit.

My concern is that this is doing quite a large amount of additional querying and is slowing down the method considerably; that might be okay, but there might be an easy way *not* to need to do it.